comp/api/api/utils/stream/stream.go

```diff
@@ -93,3 +106,70 @@ func GetStreamFunc(messageReceiverFunc func() MessageReceiver, streamType, agent
 		}
 	}
 }
+
+// ExportStreamLogs export output of stream-logs to a file. Currently used for remote config stream logs
+func ExportStreamLogs(la logsAgent.Component, streamLogParams *LogParams) error {
```

Comment:
I think this function `ExportStreamLogs ` and the `LogParams` should be part of the `comp/logs/streamlogs/impl/streamlogs.go`. It seems that this file is mostly providing utility function for streaming. The `ExportStreamLogs` is very specific of the flare and streamlogs component.